README.md

```diff
@@ -165,4 +165,12 @@ poetry install
 Finally, run the replicator:
 ```
 poetry run cdf_fabric_replicator config.yaml
+```
+
+# Docker
+
+If you are running the Docker Container on ARM based Macs. You will need to use platform emulation for the Docker run command. You can do this by running the following command:
+
+```
+docker run -i -t <image-name> --platform linux/arm64 
```

Comment:
Do we need instructions on how to build the Docker image?

Reply:
I would not think so since we would build the docker image the same way regardless of this change